tarmac/src/main/scala/nl/lumc/sasc/biopet/pipelines/tarmac/Tarmac.scala

```diff
@@ -33,7 +53,93 @@ class Tarmac(val parent: Configurable) extends QScript with PedigreeQscript with
   }
 
   def addMultiSampleJobs() = {
+    val initRefMap = samples map { x => x._2 -> getReferenceSamplesForSample(x._1) }
```

Comment:
to make this more readable, you can also pattern match on the tuple directly:
```
val initRefMap = samples.map { case (a, b) => b -> getReferenceSamplesForSample(a) }
```
(of course `a` and `b` should be named appropriately).